pkg/policyexceptions/manager.go

```diff
@@ -0,0 +1,133 @@
+package policyexceptions
```

Comment:
What is this controller doing ?

Reply:
As suggested before, this is no longer a controller. It only implements the `policyexceptions.Interface` with the ability to get exceptions by the informer.

Reply:
You shouldn't need anything like that. Why are you subscribing to informer events for example ?
You can just use a lister.